src/EnergyPlus/DataEnvironment.cc

```diff
@@ -588,6 +590,161 @@ namespace DataEnvironment {
 
 	}
 
+	Real64
+	DryBulbTempAtNode( int NodeNum )  // OutdoorAir:Node Number
```

Comment:
This function and the others added in this file should be removed and direct assignment should be used.

Reply:
Done.